pandas-stubs/_typing.pyi

```diff
@@ -42,6 +42,8 @@ from pandas._libs.tslibs import (
 
 from pandas.core.dtypes.dtypes import ExtensionDtype
 
+npt = numpy.typing
```

Comment:
why make this change.  We're following the convention in pandas of doing `from numpy import typing as npt`

Reply:
that is needed for mypy 0.971. npt needs to be re-exported

Reply:
Let me revert and see if it passes CI.